pages/agent/v3/securing.md.erb

```diff
@@ -133,3 +133,16 @@ Environment=https_proxy=http://username:password@proxyserver:8080/
 {: codeblock-file="proxy.conf"}
 
 After creating this file, systemd will require a reload and the `buildkite-agent` service will require a restart. 
+
+## Preventing Buildkite infrastructure compromise
+ 
+Suppose your Buildkite agent is being deployed into a sensitive environment. In that case, there are a few default settings that you'd want to adjust to specifically safeguard the rest of your Buildkite Organization's infrastructure in case of the agent's compromise (for example, if the agent API gets compromised).
```

Comment:
```suggestion
To safeguard your organization's infrastructure in case of Buildkite infrastructure being compromised, you want to restrict what the agent can and cannot do. All of the information above about securing the Buildkite agent applies, specifically:
```